src/components/molecules/UiLoader/UiLoader.mdx

```diff
@@ -0,0 +1,90 @@
+import { Canvas, Meta, Story, Primary, ArgsTable, Source } from "@storybook/addon-docs";
```

Comment:
@ArkadiuszKrzymienski I made a more detailed doc for UiLoader. I add `How to use` section with info about the transition types and stories for it